lib/daisybill_api/ext/crud.rb

```diff
@@ -3,6 +3,7 @@
 require 'daisybill_api/ext/crud/show'
 require 'daisybill_api/ext/crud/update'
 require 'daisybill_api/ext/crud/destroy'
+require 'daisybill_api/ext/crud/search'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.